packages/react-components/src/ChainImg.tsx

```diff
@@ -83,6 +84,6 @@ function ChainImg ({ className, logo, onClick }: Props): React.ReactElement<Prop
 }
 
 export default styled(ChainImg)`
```

Comment:
Should this icon/logo switch dynamically rather than replace 💭

Reply:
According what Ben mentioned in the comment..
The hamburger button is for mobile view when you keep narrowing down from the above view, and you would see a small black logo on the left top corner. And what we need to do is replacing that dark image to be hamburger one. We may need to make some changes around here - https://github.com/cennznet/apps/blob/develop/packages/react-components/src/ChainImg.tsx#L14